extension/README.md

```
@@ -0,0 +1,69 @@
+
+| 環境 | コマンド                         | ベース URL                |
+| ---- | -------------------------------- | ------------------------- |
+| 本番 | `bun run prepare:extension:prod` | https://zedi-note.app     |
+| 開発 | `bun run prepare:extension:dev`  | https://dev.zedi-note.app |
+
+- 本番 zip 作成前に `prepare:extension:prod` を実行してから `extension/` を zip する。
+- 開発者は `prepare:extension:dev` で dev 環境用に切り替える。
+
+## ファイル構成 / File Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)